screens/AlarmScreen.js

```diff
@@ -15,6 +15,7 @@ const AlarmScreen = () => {
 
     <Content>
       <Banner />
+      <Banner />
```

Comment:
Whats the reason for 2 banners? Same in UsingScreen line 23.

Reply:
Ideally we would put the extra icons (location, X responders available) in the second Banner

Reply:
Ahh yes. Making that icon button as a reusable component can be a separate task and we can come back to this PR later to finish it off once and for all D: